src/api/FirebaseAPI.ts

```diff
@@ -29,30 +36,13 @@ class FirebaseAPI {
   };
 
   static addFoodDetailsComponentsData = async () => {
-    Promise.all(
-      Object.entries(FoodDetailsComponentData).map(async (each) => {
-        await setDoc(doc(fStore, "FYPData", each[0]), each[1]);
-      })
+    await Promise.all(
+        Object.entries(FoodDetailsComponentData).map(async (each) => {
+          await setDoc(doc(fStore, "FYPData", each[0]), each[1]);
+        })
     );
   };
 
-  static fetchNutritionData = async (name: string):Promise<NutritionalDetailsFirebaseProps> => {
-    const querySnapshot = await getDocs(collection(fStore, "NutritionData", name , "Content"));
-
-    // @ts-ignore
```

Comment:
Is it possible to avoid the ts-ignore lines, as we have been trying to remove linting errors where possible

Reply:
yes

Reply:
Otherwise looks good to me!

Reply:
Yeah, no ignores unless necessary! If they are really needed then please add another comment explaining why.

Reply:
i have no ideaw why they were there cos there is no linting errors anyway :)

Reply:
@hannah-whelan @tom-downes-and please approve this PR when free to do so